resolver-dns/src/main/java/io/netty/resolver/dns/UnixResolverDnsServerAddressStreamProvider.java

```diff
@@ -217,7 +222,7 @@ private static void putIfAbsent(Map<String, DnsServerAddresses> domainToNameServ
                                     String domainName,
                                     List<InetSocketAddress> addresses) {
         // TODO(scott): sortlist is being ignored.
-        putIfAbsent(domainToNameServerStreamMap, domainName, DnsServerAddresses.shuffled(addresses));
+        putIfAbsent(domainToNameServerStreamMap, domainName, DnsServerAddresses.sequential(addresses));
```

Comment:
Yes this is not directly related to this change but it makes unit testing more difficult. If I pull this out I would also pull the unit tests out (and submit them together) ... is that preferred from your approach?

https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man5/resolver.5.html
```
If there are multiple servers,
     the resolver library queries them in the order listed.  The algorithm used is to try a name server, and
     if the query times out, try the next, until out of name servers, then repeat trying all the name
     servers until a maximum number of retries are made.
```